packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -115,6 +116,20 @@ contract Deploy is Deployer {
         });
     }
 
+    /// @notice Returns the proxy addresses, not reverting if any are unset.
+    function _proxiesUnstrict() private view returns (Types.ContractSet memory proxies_) {
```

Comment:
is the original `_proxies()` still in use somewhere or can it be deleted?

Reply:
The original `_proxies()` fn is still in use. This is introduced to allow for us to grab a `ContractSet` of all proxies that have been deployed so far, without erroring if any are unset.